lib/webpacker/commands.rb

```diff
@@ -7,10 +7,11 @@ def initialize(webpacker)
 
   def clean(count_to_keep = 2)
     if config.public_output_path.exist? && config.public_manifest_path.exist?
-      files_in_manifest = manifest.refresh.values.map { |f| File.join config.root_path, "public", f }
-      files_to_be_removed = files_in_manifest.flat_map do |file_in_manifest|
+      files_in_manifest = process_manifest_hash(manifest.refresh)
+      files_to_be_removed = files_in_manifest.map do |file_in_manifest|
```

Comment:
Why the switch from `flat_map` to `map`? I now see this inserting `[]` elements into `files_to_be_removed` which will throw `TypeError: no implicit conversion of Array into String` which is different but no improvement from the previous error about hashes.